argilla/src/argilla/datasets/_export/card/argilla_template.md

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
All this will need to be updated before the release

Reply:
Yep. I'm on that today.